src/components/RestaurantFilterComponent/RestaurantFilterComponent.jsx

```diff
@@ -1,260 +1,131 @@
 import { useSelector } from "react-redux";
 import "./RestaurantFilterComponent.css";
 import { useEffect } from "react";
-import { BsFillFilterSquareFill } from "react-icons/bs";
 import { useState } from "react";
-import { ImInfo } from "react-icons/im";
 import { useSearchParams } from "react-router-dom";
 import languageSelector from "../../assets/languages/languageSelector";
+import burger from "../../assets/hamburger.svg";
+import CheckboxFilter from "./CheckboxFilter";
+import cuisinesDB from "../../assets/cuisines.json";
 
-const RestaurantFilterComponent = ({
-  categories,
-  setCategories,
-  rating,
-  setRating,
-  data,
-}) => {
+const RestaurantFilterComponent = () => {
+  const cuisines = Object.keys(cuisinesDB);
   const [searchParams, setSearchParams] = useSearchParams({});
 
-  const [mobileFilter, setMobileFilter] = useState("");
   const [radioSelected, setRadioSelected] = useState("");
-  const [checkBoxSelected, setCheckBoxSelected] = useState([]);
+  const [mobileView, setMobileView] = useState(false);
 
   const language = useSelector((state) => state.languageReducer);
 
+  //the existence of a rating type filter in the url is verified, if so, the corresponding value is applied to the state of the radio buttons so that it is shown selected
```

Comment:
```
/**
the existence of a rating type filter in the url is verified, if so, the corresponding value is applied to the state of the radio buttons so that it is shown selected
*/
```